tools/clang-tidy-plugin/test/lit.site.cfg.in

```diff
@@ -2,7 +2,6 @@ import os
 
 config.cata_source_dir = "@CMAKE_SOURCE_DIR@"
 config.plugin_build_root = "@CMAKE_CURRENT_BINARY_DIR@"
-config.cata_check_clang_tidy = "@CATA_CHECK_CLANG_TIDY@"
```

Comment:
This is needed on Windows to add custom executable path argument to the `check_clang_tidy.py` call.

Reply:
Are you saying that on Windows you need a different `check_clang_tidy.py`, or that you need to pass different arguments to `check_clang_tidy.py`?

Reply:
Both at the moment. I think I can switch to using `check_clang_tidy.py` in this PR but even then a different argument is needed, and I'll need to add my patches for it to work.

Reply:
I have mixed feelings about having custom patches to the in-repo `check_clang_tidy.py`.  On the one hand it's convenient.  On the other it will make it annoying to incorporate upstream changes to the file.

For now I guess I'll put these features back in so they're available to Windows folks if they need them.